daml/Marketplace/Issuer.daml

```diff
@@ -49,7 +49,8 @@ template Issuer
           observers : [Party]
         do
           let tokenId = Id with signatories = fromList [issuer], label = name, version = 0
-          create Token with id = tokenId, observers = if isPublic then fromList [public] else fromList observers, ..
+          let tokenObservers = if isPublic then (public :: observers) else observers
```

Comment:
if it is public doesnt that mean all are observers anyway?

Reply:
The only reason I changed this is because the original DAML didn't seem to work right when `isPublic` was false. The observers would get passed in, but the final Token contract didn't include them. 

So I refactored the line a little bit. Not sure why this works works when the original didn't, though